releasenotes/notes/add-peer-tags-configuration-79529a2af74e655a.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adds `peer_tags` configuration to allow for more tags in APM stats that
```

Comment:
Please add more details here. For example, how should it be configured? You can use this as an example: https://github.com/DataDog/datadog-agent/blob/main/releasenotes/notes/apm-memcached-keep-command-aecd5105a09e1235.yaml